src/config/configFile/ensureHomeDirFactory.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+
+const CouldNotCreateHomeDirError = require('../errors/CouldNotCreateHomeDirError')
+
+/**
+ * @param {string} homeDirPath
+ * @return {ensureHomeDir}
+ */
+function ensureHomeDirFactory(homeDirPath = path.resolve(os.homedir(), '.mn')) {
```

Comment:
I would remove the default value for the dependency. It should be defined in the container.